frontend/js/main.js

```diff
@@ -157,16 +109,21 @@ const createEditDataTypeTable = () => {
           selectHTML += `<option value='${globalDataTypeList[Object.keys(globalDataTypeList)[i]][k].T}'>${globalDataTypeList[Object.keys(globalDataTypeList)[i]][k].T} </option>`;
         }
         selectHTML += `</select>`;
```

Comment:
1. selectHTML is assigned two times same html, check the line no 98 to 101 and its same for 104 to 107
2. also line no 101 and 107, </div> should not be there inside <select> tag

Reply:
1. selectHTML is almost same but only difference is 'visibility: hidden' property of material icon in <i> tag. This was needed according to UI. May be we can connect for this and I can explain you by pointing in the UI

Reply:
we can build <i> tag in selectHTML as per the condition instead of repeating complete selectHTML section

Reply:
this change is done

Reply:
@13shreya Please check its still the same code, no change found here